storm-core/src/native/worker-launcher/impl/main.c

```diff
@@ -269,6 +274,45 @@ int main(int argc, char **argv) {
       return INVALID_ARGUMENT_NUMBER;
     }
     exit_code = signal_container_as_user(user_detail->pw_name, container_pid, signal);
+  } else if (strcasecmp("run-oci-container", command) == 0) {
+    if (argc != 6) {
+      fprintf(ERRORFILE, "Incorrect number of arguments (%d vs 6) for run-oci-container\n", argc);
+      fflush(ERRORFILE);
+      return INVALID_ARGUMENT_NUMBER;
```

Comment:
nit: log and error files are closed before return for most cases. set exit_code here also?

Reply:
I don't understand the question. Can you please elaborate?

Reply:
Set the exit code to non-zero so that it is returned at the end of this function (where it properly closes open files). Although I don't think it is critical, since open files will be closed by the system on program exit.

Reply:
got it. Thanks